HACKING.md

```diff
@@ -34,3 +34,7 @@ Some rules support multiple languages, like `____func_code____`, which generates
 A relatively easy way to add a feature in Cognac is to start by modifying the generated code (such as "osc-sdk.c", "main.c", etc.). First, make your changes directly in the generated code, ensure it works as expected, and then modify the generator accordingly. This approach helps you know exactly what to look for in the generator.
 
 For exemple, if you modify a function like `parse_thatarg()` in "osc_sdk.c", once the changes are working, you can then search (using a tool like grep) through the generator code to find where `parse_thatarg` is generated. From there, you can add modifications to the generator to make the change permanent and automated.
+
+## helpers.sh
+
+most function in helper.sh have been documented, so go read it if you want to understand some of the most used functions of cognac_gen
```

Comment:
Most functions in helper.sh are documented, so read through it if you want to understand some of the most commonly used functions in cognac_gen.